content/applications/finance/fiscal_localizations/argentina.rst

```diff
@@ -948,6 +948,77 @@ the corresponding layout and fiscal stamps stated in the :ref:`Invoice printed r
 .. seealso::
    :doc:`Client account creation <../../websites/ecommerce/checkout_payment_shipping/checkout>`
 
+Liquidity Product Direct Sales
+----------------------------------------
+
+The liquidity product direct sale is used, for example, in the cases where sales are made by 
+the order of a third-party.
+
+For these cases, the seller and the proprietary company of the goods, will be able to register 
+their corresponding sales and purchase. 
+
+This feature is included when installing the Argentinian EDI module (l10n_ar_edi). To install 
+or update this module, go to :guidable:’Apps > Remove filters > Search ‘l10n_ar_edi’’.
+
+.. image:: ecuador/l10n_ar_edi.png
+   :align: center 
+   :alt: l10n_ar EDI Module
```

Comment:
Can reference the Install documentation and no need to include an image of the module on the Apps screen :)
```suggestion
This feature is included when :ref:`installing <general/install>` the *Argentinian EDI* module
(`l10n_ar_edi`).
```